### DISCUSSION TITLE
Inline Link

### DISCUSSION THREAD

**Discussion by rshov** [heroui-inc/heroui#726] [Help] [answered]
It doesn't seem like there is an option to make the `<Link>` component display inline. 
Seems like that should be the default functionality. Am I missing something?

The `block` attribute defaults to false according to the docs, but even setting `<Link block={false}>` puts the link on a new line.

https://codesandbox.io/s/sandpack-project-forked-ptkwdr

**Comment by jrgarciadev** [comment, marked as answer]
Hey @rshov, this was fixed here https://github.com/nextui-org/nextui/commit/2e00416d17d7b55c11dc9688e1d711a039504f56, however, It will be available in the next version (v11), in the meanwhile you can fix it as follows:

```jsx
<Link
  href="#"
  css={{
    d: "inline-flex"
  }}
>
  My Link
</Link>
```

e.g. https://codesandbox.io/s/sandpack-project-forked-vo2h2d?file=/App.js

**Reply by rshov** [reply]
@jrgarciadev  Perfect, thank you!
